cyberdrop_dl/main.py

```diff
@@ -71,13 +90,19 @@ async def director(manager: Manager) -> None:
     if cyberdrop_dl.utils.utilities.DEBUG_VAR:
         logger_debug.setLevel(manager.config_manager.settings_data['Runtime_Options']['log_level'])
         if os.getenv("PYCHARM_HOSTED") is not None or 'TERM_PROGRAM' in os.environ.keys() and os.environ['TERM_PROGRAM'] == 'vscode':
-            file_handler_debug = logging.FileHandler("../cyberdrop_dl_debug.log", mode="w")
+            debug_log_file_path = Path(__file__).parents[1] / "cyberdrop_dl_debug.log"
```

Comment:
Would this work properly when built with Poetry? `__file__` would reference the file's path rather than the current working directory causing logs to be in the user's python folder no?

Reply:
Nevermind it wouldn't produce the `cyberdrop_dl_debug.log` file when running on a build from pypi. All looks good.